examples/basic-tracer/tracer.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from opentelemetry import trace
+from opentelemetry.context import Context
+from opentelemetry.ext.jaeger import JaegerSpanExporter
```

Comment:
I think this import has to be moved under the conditional because somebody trying only the console exporter will not have installed the Jaeger one.
Applies for other examples as well.

Reply:
good point, updated the examples

Reply:
@codeboten It seems that the import is still here though. Did you push your changes?